src/metabase/config.clj

```diff
@@ -23,6 +23,8 @@
    ;; Jetty Settings. Full list of options is available here: https://github.com/ring-clojure/ring/blob/master/ring-jetty-adapter/src/ring/adapter/jetty.clj
    :mb-jetty-port          "3000"
    :mb-jetty-join          "true"
+   :mb-jetty-max-request-header-size-bytes  "8192"
+   :mb-jetty-max-response-header-size-bytes "8192"
```

Comment:
I did a final live test and while running this code I realized that these defaults should probably be removed: if the defaults are specified here they will always override Ring's defaults. In other words, if the Ring defaults should change in the future, these defaults will represent an unexpected deviation. Instead it's better to remove these defaults, so that the values are only set when the relevant `MB_JETTY_*` environment variable is set.

TL;DR: Just remove these lines :stuck_out_tongue:. The rest is great.

```suggestion
```